### ISSUE TITLE
Improves api consistency related to modules containing the generated protobuf assets

### ISSUE THREAD

**Issue by tottoto** [hyperium/tonic#1221]
### Crates

- tonic-types
- tonic-health
- tonic-reflection

### Motivation

Improves apis consistency related to modules containing the generated protobuf assets.

### Proposal

> **Warning**: This proposal includes breaking changes.

In this discussion, I assume the project status of 0ca0630993beab8c17e14049e30887d4abd59d36. Currently we have some inconsistency apis in these crates.

|  | tonic-types | tonic-health | tonic-reflection |
| --- | --- | --- | --- |
| module name of protobuf | pb | proto | proto |
| variable name of encoded file descriptor set | FILE_DESCRIPTOR_SET | GRPC_HEALTH_V1_FILE_DESCRIPTOR_SET | FILE_DESCRIPTOR_SET |

Regarding to the module name of protobuf, my proposal is to rename all module names to pb or proto. My preference is pb as pb might better represent protobuf. proto is a common word so I think pb is a more appropriate module name.

When we uses the encoded file descriptorset (for example: `FILE_DESCRIPTOR_SET` in tonic-types), we uses [`encoded_file_descriptor_set`](https://docs.rs/tonic-reflection/0.6.0/tonic_reflection/server/struct.Builder.html#method.register_encoded_file_descriptor_set). However there is [`register_file_descriptor_set`](https://docs.rs/tonic-reflection/0.6.0/tonic_reflection/server/struct.Builder.html#method.register_file_descriptor_set). these apis might be confusable. In this point, my proposal is rename the current variable name of the encoded file descriptor set to `ENCODED_FILE_DESCRIPTOR_SET` and provide new `FILE_DESCRIPTOR_SET` which is the [`FileDescriptorSet`](https://docs.rs/prost-types/0.11.5/prost_types/struct.FileDescriptorSet.html).

### Alternatives

Leaves these apis inconsistency. However we might be able to take these proposals partially. For example, renaming `GRPC_HEALTH_V1_FILE_DESCRIPTOR_SET` in tonic-health to `FILE_DESCRIPTOR_SET` can be introduced without breaking changes by just being added as a new variable.

**Comment by LucioFranco** [comment]
Yeah, I think changing them all to use `pb` and rename the FDS to be `FILE_DESCRIPTOR_SET` is also a good change. Both seem good to me 👍

**Comment by tottoto** [comment]
Thanks for your review! I'll work on this.

**Comment by tottoto** [comment]
I noticed that we can not get `prost_types::FileDescriptorSet` as static variable since `prost::Message::decode` is a non-const fn. So I'll change all file descriptor set variable names to `FILE_DESCRIPTOR_SET` which contains byte encoded value.
